src/CoreTests/CoreTests.csproj

```diff
@@ -87,11 +87,6 @@
     <Compile Include="Bases\A11yPropertyTests.cs" />
     <Compile Include="Fingerprint\ScanResultFingerprintUnitTests.cs" />
     <Compile Include="Fingerprint\FingerprintContributionUnitTests.cs" />
-    <Compile Include="Fingerprint\IssueUnitTests.cs" />
```

Comment:
>IssueUnitTests [](start = 34, length = 14)

These are the tests that correspond to the classes removed from Core\Fingerprint